test/blocks/global-footer/test-utilities.js

```diff
@@ -0,0 +1,97 @@
+import { setViewport } from '@web/test-runner-commands';
+import { setConfig } from '../../../libs/utils/utils.js';
+import { config, isElementVisible, loadStyles, viewports } from '../global-navigation/test-utilities.js';
+import { waitForElement } from '../../helpers/waitfor.js';
+import { selectors as keyboardSelectors } from '../../../libs/blocks/global-navigation/utilities/keyboard/utils.js';
+import { selectors as gnavSelectors } from '../../../libs/blocks/global-navigation/utilities/utilities.js';
+
+export const allSelectors = {
+  container: keyboardSelectors.globalFooter,
+  footerIcons: '.feds-footer-icons',
+  footerWrapper: '.feds-footer-wrapper',
+  menuContent: keyboardSelectors.menuContent,
+  menuColumn: gnavSelectors.menuColumn,
+  menuSection: gnavSelectors.menuSection,
+  menuHeadline: '.feds-menu-headline',
+  menuItems: '.feds-menu-items',
+  featuredProducts: '.feds-featuredProducts',
+  featuredProductsLabel: '.feds-featuredProducts-label',
+  navLink: gnavSelectors.navLink,
+  navLinkImage: '.feds-navLink-image',
+  navLinkContent: '.feds-navLink-content',
+  navLinkTitle: '.feds-navLink-title',
+  social: '.feds-social',
+  socialItem: '.feds-social-item',
+  regionPickerWrapper: '.feds-regionPicker-wrapper',
+  regionPicker: keyboardSelectors.regionPicker,
+  legalWrapper: '.feds-footer-legalWrapper',
+  privacySection: '.feds-footer-privacySection',
+  copyright: '.feds-footer-copyright',
+  privacyLink: keyboardSelectors.privacyLink,
+};
+
+const getMobileVisibleSelectors = () => {
+  const {
+    container,
+    footerIcons,
+    featuredProducts,
+    featuredProductsLabel,
+    navLink,
+    navLinkImage,
+    navLinkContent,
+    navLinkTitle,
+    menuItems,
+    menuHeadline, ...mobileSelectors
```

Comment:
Could we move `...mobileSelectors` to  a new line? Does ESLint not complain because of the lack of a final comma?

Reply:
Missed that one, weird that ESLint did not complain